blivet/partitioning.py

```diff
@@ -350,7 +350,7 @@ def remove_new_partitions(disks, remove, all_partitions):
         # remove empty extended so it doesn't interfere
         extended = disk.format.extended_partition
         if extended and not disk.format.logical_partitions and \
-           (flags.installer_mode or
+           (flags.force_dev_updates or
```

Comment:
Please make a separate flag for this -- maybe `keep_empty_extended_partition`. Note the sense would be the reverse of `installer_mode` above since we keep empty pre-existing extended partitions except when anaconda is driving.